phpstan-baseline.neon

```diff
@@ -0,0 +1,65 @@
+
+
+parameters:
+	ignoreErrors:
+		-
+			message: "#^Call to an undefined method DOMNode\\:\\:setAttribute\\(\\)\\.$#"
```

Comment:
Is it because of the dom extension missing on the phpqa image?

Reply:
Probably not. Had this inconsistency on another project as well. Typehinting it as `DOMElement` instead solves this error.

The object actually isn't a `DOMNode`. This is because of the return signature on `appendChild` in PHP core.

Reply:
Btw, would be great to document how to use jakzal/phpqa with github actions. I'd appreciate a PR if you have some spare time. Otherwise I'll try to do it in the next couple of weeks.

Reply:
Will send you a PR once this one is ready 👍 

Locally the baseline file gets used. Any idea why it's not used by Github checks?

Reply:
Make sure you're testing with the latest image locally. PHPStan was recently updated.